packages/components/src/components/clipboard/clipboard.jsx

```diff
@@ -19,24 +19,20 @@ const Clipboard = ({
     const isMounted = useIsMounted();
     let timeout_clipboard = null;
 
-    const copyToClipboard = text => {
-        const textField = document.createElement('textarea');
-        textField.innerText = text;
-        document.body.appendChild(textField);
-        textField.select();
-        document.execCommand('copy');
-        textField.remove();
+    const copyToClipboard = async text => {
+        await navigator.clipboard.writeText(text);
```

Comment:
@Bahar-FS How about we extract this into a `useClipboard` hook in another PR?

Reply:
@farzin-fs Good suggestion, I'll create another card for it so we can refactor other places using the deprecated one with it.